lib/bitstyles_phoenix/live/sidebar.ex

```diff
@@ -64,7 +64,6 @@ defmodule BitstylesPhoenix.Live.Sidebar do
           JS.hide(to: "##{id}", transition: {"is-transitioning", "is-on-screen", "is-off-screen"}),
         "aria-controls": id
       )
-      |> Keyword.put_new(:reversed, true)
```

Comment:
⚠️ this is also a breaking change and probably needs some extra code for making it backwards compatible. See the respective section for the release of version 5.0.0 in [bitstyles' changelog](https://github.com/bitcrowd/bitstyles/blob/465157560507997b55c5d796b0cec90e30906a0f/CHANGELOG.md).

Not sure how the sidebar looks without that button type though 🤔